source/standards/running-services-as-containers.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: How to run your service as a container
+
+expires: 2018-05-02
+---
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+If your web application is running on [GOV.UK Platform as a Service (PaaS)][] you should follow the [recommended hosting guidance][].
+
+If your web application is running on PaaS you should use:
+
+* [Docker][] to [build and deploy images][], as it’s widely used across GDS and industry
+* [Dockerhub][] or [Amazon EC2 Container Registry][] to host your images following the [PaaS responsibility model][]
```

Comment:
This contradicts the current advice from the PaaS team. Docker support is currently experimental, and not recommended for production services. The current implementation of Docker support in Cloudfoundry has some limitations that mean we don't consider it suitable for production use - https://docs.cloud.service.gov.uk/#deploy-a-docker-image-experimental.

The most significant of these is that it it ties the availability of the application to the availability of the Docker registry being used - if the registry is unavailable, your application could fail at any time.

If deploying to the PaaS we strongly recommend using the buildpack approach over Docker images.